keras_nlp/tests/doc_tests/docstring_test.py

```diff
@@ -13,17 +13,27 @@
 # limitations under the License.
 
 import doctest
+import io
 import os
 import sys
 import unittest
 
 import numpy as np
 import pytest
+import sentencepiece
 import tensorflow as tf
 from tensorflow import keras
 
 import keras_nlp
 from keras_nlp.tests.doc_tests import docstring_lib
+from keras_nlp.tests.doc_tests import fenced_docstring_lib
+from keras_nlp.tests.doc_tests.fenced_docstring_lib import (
+    astor,  # For checking conditional import.
+)
+
+ASTOR_WARNING = (
+    "`test_fenced_docstrings` requires `astor`. Please run `pip install astor`."
```

Comment:
We don't usually keep warning string in constant like this, I'm just going to inline this.